Assets/VRTK/SDK/HyperealVR/SDK_HyperealVRDefines.cs

```diff
@@ -0,0 +1,28 @@
+﻿// HyperealVR Defines|SDK_HyperealVR|001
+namespace VRTK
+{
+    using System;
+    using System.Reflection;
+
+    /// <summary>
+    /// Handles all the scripting define symbols for the Hypereal SDK.
+    /// </summary>
+    public static class SDK_HyperealVRDefines
+    {
+        /// <summary>
+        /// The scripting define symbol for the Hypereal SDK.
+        /// </summary>
+        public const string ScriptDefineSymbol = SDK_ScriptingDefineSymbolPredicateAttribute.RemovableSymbolPrefix +
+            "SDK_HYPEREALVR";
+
+        private const string BuildTargetGroupName = "Standalone";
+
+        [SDK_ScriptingDefineSymbolPredicate(ScriptDefineSymbol, BuildTargetGroupName)]
+        [SDK_ScriptingDefineSymbolPredicate(SDK_ScriptingDefineSymbolPredicateAttribute.RemovableSymbolPrefix +
```

Comment:
No need for this attribute when you don't have multiple versions to add scripting define symbols for _in addition to the general SDK symbol_. So I suggest to remove this line. (Also see the next comment.)